run.py

```diff
@@ -591,9 +712,12 @@ def run_replanning_loop(settings, forecast_year):
             if forecast_year == year:
                 warm_start_skims = True
 
-            generate_activity_plans(
-                settings, year, forecast_year, client,
-                warm_start=warm_start_skims)
+            if demand_model == 'polaris':
```

Comment:
@jaauld14 again I think it would be cleaner if you moved this conditional logic into the main `generate_activity_plans()` method and pass in `demand_model` as an argument.

Reply:
Done